platform/ui/src/components/studyList/StudyList.styl

```diff
@@ -76,14 +100,106 @@ placeholder-color(c)
     height: 1px
     z-index: 2
 
-#studyListContainer
+
+.study-list-container
   width: 100%
   padding: 0 $study-list-padding
   position: absolute
   z-index: 2
 
+table.table
+  width: 100%;
+  margin-bottom: 20px;
+  border-spacing: 0;
+  border-collapse: collapse;
+  table-layout: fixed;
+
+  > tbody tr
+    padding: 5px
+    background-color: black
+
+  /* Striped Variant */
+  &.table--striped > tbody tr:nth-child(even)
+    background-color: var(--ui-gray-darker)
+
+  /* Hover Variant */
+  &.table--hoverable > tbody tr
+    &:hover, &:active, &.active
+      background-color: var(--table-hover-color)
+      color: var(--text-primary-color)
+
+.study-list-container > table.table > tr
+  height: 20px
+
+.study-list-container > table.table > thead
+  ::-webkit-datetime-edit-year-field:not([aria-valuenow]),
+  ::-webkit-datetime-edit-month-field:not([aria-valuenow]),
+  ::-webkit-datetime-edit-day-field:not([aria-valuenow])
+    color: transparent
+
+.study-list-container > table.table > thead > tr > th
+  padding: 0;
+  border-bottom: 1px solid var(--ui-border-color-active);
+  width: 100%;
+  text-align: left;
+  border-top: 0;
+
+.study-list-container > table.table > tbody > tr > td
+  padding: $body-cell-top-bottom-padding 8px;
+  height: $body-cell-height
+  color: var(--table-text-primary-color)
+  font-weight: 300
+  word-wrap: break-word;
+
+  &.emptyCell
+    color: var(--ui-gray-light)
+
+.study-list-container > table.table > thead > tr > th.studyDate
+  min-width: 230px
+
+.study-list-container
+  .filters
+    label
+      display: flex;
```

Comment:
![image](https://user-images.githubusercontent.com/1905961/67530253-ea6b3d00-f694-11e9-9c5f-c42efbe7bcd2.png)


The sorting arrow is a little bit misaligned, you can fix that by adding `align-items: center;`:

```suggestion
      display: flex;
      align-items: center;
```

Reply:
Ooh. Using GitHub `suggestions` feature.
Thanks for the tip 🙏